test/common_nn.py

```diff
@@ -322,11 +322,55 @@ def smoothl1loss_reference(input, target, size_average=True, reduce=True):
     return output
 
 
+def _multilabelmarginloss_reference(input, target, is_target):
```

Comment:
nit: You can nest this function inside of `multilabelmarginloss_reference` so that it is only visible to the scope of that function but it doesn't really matter